modules/ocf_hpc/manifests/singularity.pp

```diff
@@ -0,0 +1,22 @@
+class ocf_hpc::singularity {
+
+  package { 'singularity-container':
+    install_options => ['-t stretch-backports'],
+  }
+
+  file { '/etc/singularity/singularity.conf':
+    source => 'puppet:///modules/ocf_hpc/singularity.conf',
+    mode   => '0644',
+    owner  => 'root',
+    group  => 'root',
+  }
+
+  # Each running singularity container consumes a loop device,
+  # and the default max loop devices is 8.
+  # Requires reboot after change.
+  augeas { 'modules':
```

Comment:
Consider naming this resource something like `Set singularity max loop devices`. Resource names must be globally unique across all types, and `modules` is an awfully generic name.